salad/operator/scripts/wait_for_epoch_transition.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const Web3 = require('web3');
+const {Enigma} = require('enigma-js/node');
+
+const main = async () => {
+    const provider = new Web3.providers.WebsocketProvider(`ws://${process.env.ETH_HOST}:${process.env.ETH_PORT}`);
```

Comment:
Could this break in real networks where the websocket port is not the same as the http port? For example, in our testnet http is 8545 and WS is 8546

Reply:
Yes, you're right...
Is there a reason these have to be different?
I can of course add another env variable which would be different where that's relevant.
Keep in mind that this script should not run in testnet/mainnet.

Reply:
Different services on different ports, it makes sense. 

But if you're only running it in k8s/compose then you can leave it like this, it will be fine.

Reply:
I think i'll leave it like this for now. when it becomes relevant it'll be an easy fix